research-software-directory/RSD-as-a-service
# Create a fake data generation script

# Data generation

Changes proposed in this pull request:

* A Node.js script is created that can generate fake data for the RSD. This data can locally be used for developing and testing purposes.

How to test:
* `docker-compose down --volumes && docker-compose build --parallel && docker-compose up --scale data-generation=1 --scale scrapers=0`
* Visit http://localhost and admire all the fake data
* Check if anything is missing

Closes #473

PR Checklist:
